YOLayout/YOLayout_IMP.m

```diff
@@ -138,11 +138,11 @@ - (CGRect)setSize:(CGSize)size inRect:(CGRect)inRect view:(id)view options:(YOLa
     size = sizeThatFits;
   }
   
-  if ((options & YOLayoutOptionsSizeToFitHorizontal) == 0) {
+  if ((options & YOLayoutOptionsSizeToFitHorizontal) == 0 && originalSize.width > 0) {
```

Comment:
So this sizes to fit even when not passing in `YOLayoutOptionsSizeToFitHorizontal` when the original size is 0?